### ISSUE TITLE
Calendar Entity "color" stopped working in 4.4.0

### ISSUE THREAD

**Issue by ejpenney** [marksie1988/atomic-calendar-revive#452]
## Expected behaviour
Atomic Calendar card should be configurable like so as documented in examples:
```
entities:
  - entity: calendar.MY_CALENDAR
    color: 'red'
```

## Current behaviour
In 4.4.0+ all calendar entries appear white

## Screenshot
4.4.0:
![image](https://user-images.githubusercontent.com/21226768/123124099-3b987a80-d3fc-11eb-9ba7-27ffcb5dd189.png)
Reverted to 4.3.0:
![image](https://user-images.githubusercontent.com/21226768/123124241-5a970c80-d3fc-11eb-9a2e-85b5149d4d4c.png)

## Possible reasons and solutions
I compared tags and don't see any obvious reasons for this regression, but can state that reverting to 4.3.0 always fixes the issue.

## Steps to reproduce
1. Go to Dashboard
2. Add Atomic Calendar Card
3. Set Calendar Entity color
4. Save and load dashboard, dashboard appears white and ignores color

## Context
Colors on the calendar are useful for at-a-glance usage.  My color is red, I can glance at the calendar and see red items, other family members feel the same.

## Custom values
The rest of the card is configured as:
```
enableModeChange: true
defaultMode: Calendar
disableEventLink: true
disableLocationLink: true
disableCalEventLink: false
disableCalLocationLink: true
disableCalLink: false
maxDaysToShow: 7
firstDayOfWeek: 0
```

## Your environment
![image](https://user-images.githubusercontent.com/21226768/123125341-49023480-d3fd-11eb-9173-8de6b07f0195.png)
Atomic Calendar is installed via HACS, using default theme.  Additionally I'm running these on the frontend:

- layout-card/2.2.3
- Kiosk mode/1.6.5
- card-tools/1.1
- Lovelace Google Keep Card/1.2.0
- mini climate card/1.2.7
- Lovelace Swipe Navigation/1.3.5
- Lovelace Xiaomi Vacuum Map Card/1.2.3

## Stack trace / logging
I don't see any errors in Chrome Console.

## Checklist

- [x] I searched the current issues and the issue is not known.
- [x] I searched the troubleshooting and there's no fix available for this bug.
- [x] I searched on Google, YouTube, Reddit and there's no fix.
- [] I am mostly sure that the problem is related to the code and isn't my fault.

**Comment by Ylinku** [comment]
same issue here.

**Comment by marksie1988** [comment]
Thanks, I also see this issue. Only applies to the calendar mode and is working on the event mode.

**Comment by ejpenney** [comment]
Thanks, can confirm it's fixed in 4.4.2!
